llvm/lib/Transforms/Instrumentation/MemorySanitizer.cpp

```diff
@@ -4033,6 +4041,7 @@ struct MemorySanitizerVisitor : public InstVisitor<MemorySanitizerVisitor> {
 
   void visitIntrinsicInst(IntrinsicInst &I) {
     switch (I.getIntrinsicID()) {
+#if 0
```

Comment:
this seems wrong.

Reply:
Thanks, this patch adds debugging but I accidentally left in too much debugging.